Makefile

```diff
@@ -321,6 +321,7 @@ generate-dockerfiles:
 	rm -rf /tmp/serverless-operator-generator
 	$(shell go env GOPATH)/bin/generate \
 		--generators dockerfile \
+		--dockerfile-image-builder-fmt "registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.22-openshift-4.17%.0s"  \
```

Comment:
`%s.0` is truncating string formatting to zero length, otherwise there's an extra param error in the result.

Reply:
We could probably fix it somehow in openshift-knative/hack and allow passing just string in which case it would be used as is.